### PR TITLE
[CLOUDTRUST-2320] Use default value for admin conf. Validation enhancements

### PR DESCRIPTION
**Pull request by fperot74**
Needs cloudtrust/common-service#42

### CODE REVIEW

File: api/management/api.go [hunk 1]
```diff
@@ -525,9 +538,43 @@ func (config RealmCustomConfiguration) Validate() error {
 }
 
 // Validate is a validator for RealmAdminConfiguration
-func (config RealmAdminConfiguration) Validate() error {
+func (rac *RealmAdminConfiguration) Validate() error {
```
Comment (harture) [inline api/management/api.go]: Why a pointer ?

File: api/management/api.go [hunk 2]
```diff
@@ -525,9 +538,43 @@ func (config RealmCustomConfiguration) Validate() error {
...
+					}
+				}
+			}
+			return nil
+		}).
+		Status()
+}
+
+// Validate is a validator for RealmAdminAccreditation
+func (acc *RealmAdminAccreditation) Validate() error {
```
Comment (harture) [inline api/management/api.go]: Really over a pointer?

File: api/management/api.go [hunk 3]
```diff
@@ -525,9 +538,43 @@ func (config RealmCustomConfiguration) Validate() error {
...
 	return validation.NewParameterValidator().
-		ValidateParameterRegExp("mode", config.Mode, RegExpName, true).
+		ValidateParameterLongDuration("validity", acc.Validity, true).
```
Comment (harture) [inline api/management/api.go]: ValidateParameterStringDuration ?

Comment (fperot74) [reply api/management/api.go]: Ok... ValidateParameterLargeDuration

**Pull request merged**